flutter_local_notifications/lib/src/platform_specifics/android/enums.dart

```diff
@@ -262,3 +262,82 @@ enum NotificationVisibility {
+  static const AudioAttributesUsage assistanceNavigationGuidance =
+      AudioAttributesUsage._(12);
+
+  /// Corresponds to [`AudioAttributes.USAGE_ASSISTANCE_SONIFICATION`](https://developer.android.com/reference/android/media/AudioAttributes#USAGE_ASSISTANCE_SONIFICATION).
+  static const AudioAttributesUsage assistanceSonification =
+      AudioAttributesUsage._(13);
+
+  /// Corresponds to [`AudioAttributes.USAGE_ASSISTANT`](https://developer.android.com/reference/android/media/AudioAttributes#USAGE_ASSISTANT).
+  static const AudioAttributesUsage assistant = AudioAttributesUsage._(16);
+
+  /// Corresponds to [`AudioAttributes.USAGE_GAME`](https://developer.android.com/reference/android/media/AudioAttributes#USAGE_GAME).
+  static const AudioAttributesUsage game = AudioAttributesUsage._(14);
+
+  /// Corresponds to [`AudioAttributes.USAGE_MEDIA`](https://developer.android.com/reference/android/media/AudioAttributes#USAGE_MEDIA).
+  static const AudioAttributesUsage media = AudioAttributesUsage._(1);
+
+  /// Corresponds to [`AudioAttributes.USAGE_NOTIFICATION`](https://developer.android.com/reference/android/media/AudioAttributes#USAGE_NOTIFICATION).
+  static const AudioAttributesUsage notification = AudioAttributesUsage._(5);
+
+  /// Corresponds to [`AudioAttributes.USAGE_NOTIFICATION_EVENT`](https://developer.android.com/reference/android/media/AudioAttributes#USAGE_NOTIFICATION_EVENT).
+  static const AudioAttributesUsage notificationEvent =
+      AudioAttributesUsage._(10);
+
+  /// Corresponds to [`AudioAttributes.USAGE_NOTIFICATION_RINGTONE`](https://developer.android.com/reference/android/media/AudioAttributes#USAGE_NOTIFICATION_RINGTONE).
+  static const AudioAttributesUsage notificationRingtone =
+      AudioAttributesUsage._(6);
+
+  /// Corresponds to [`AudioAttributes.USAGE_UNKNOWN`](https://developer.android.com/reference/android/media/AudioAttributes#USAGE_UNKNOWN).
+  static const AudioAttributesUsage unknown = AudioAttributesUsage._(0);
+
+  /// Corresponds to [`AudioAttributes.USAGE_VOICE_COMMUNICATION`](https://developer.android.com/reference/android/media/AudioAttributes#USAGE_VOICE_COMMUNICATION).
+  static const AudioAttributesUsage voiceCommunication =
+      AudioAttributesUsage._(2);
+
+  /// Corresponds to [`AudioAttributes.USAGE_VOICE_COMMUNICATION_SIGNALLING`](https://developer.android.com/reference/android/media/AudioAttributes#USAGE_VOICE_COMMUNICATION_SIGNALLING).
+  static const AudioAttributesUsage voiceCommunicationSignalling =
+      AudioAttributesUsage._(3);
+
+  /// All the possible values for the [AudioAttributesUsage] enumeration.
+  static List<AudioAttributesUsage> get values => <AudioAttributesUsage>[
+        alarm,
+        assistanceAccessibility,
+        assistanceNavigationGuidance,
+        assistanceSonification,
+        assistant,
+        game,
+        media,
+        notification,
+        notificationEvent,
```

Comment:
Was about to merge and then realised `notificationRingtone` is missing here. Are you able to add this in?